[Document Translation] Removing the word All from GetAll [Translation/Document] Statuses

AhmedLeithy opened the pull request:
GetAllTranslationStatuses --> GetTranslationStatuses
GetAllDocumentStatuses --> GetDocumentStatuses

This is in response to the architects' comments as per #23088 .

AhmedLeithy committed:
Removed the word All from GetAllTranslationStatuses and GetAllDocumentStatuses

AhmedLeithy committed:
Updated NetStandard

AhmedLeithy committed:
Updated Snippets

AhmedLeithy committed:
Reran some tests and renamed some SessionRecords to ensure consistency with test names

maririos commented on sdk/translation/Azure.AI.Translation.Document/src/DocumentTranslationClient.cs:
```diff
@@ -219,11 +219,11 @@ public virtual async Task<DocumentTranslationOperation> StartTranslationAsync(Do
         /// </summary>
         /// <param name="filter">Options to use when filtering result.</param>
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
-        public virtual Pageable<TranslationStatus> GetAllTranslationStatuses(TranslationFilter filter = default, CancellationToken cancellationToken = default)
```
consider updating the docstring too, so we don't say `results for all submitted translation...` and instead say `results for submitted translations`. 
Or we could say `Get the status results for all submitted translation operations according to the values of the filters"

maririos replied:
same comment will apply for the Documents methods

AhmedLeithy replied:
Yea sure. I went with the former suggestion because I felt like it is clearer and more straightforward. I noticed that we also won't be calling them filters anymore so maybe I can change it to : 

`Get the status results for all submitted translation operations according to the options provided`

but this may imply that options **need** to be provided, so I just chose to simply remove the word all. What do you think?

maririos replied:
sounds good!

maririos reviewed (commented):
curious: I see some recorded test with renames and other that were deleted and generated again. I wonder why the combination?
It might be better to just do the renames on the files so it is easier to review that nothing else changed

AhmedLeithy committed:
Revert "Reran some tests and renamed some SessionRecords to ensure consistency with test names"

This reverts commit 0d96185fa1b7aa88ece308e88e2b8e4fbce28609.

AhmedLeithy committed:
Renamed session records

AhmedLeithy committed:
Removed the word all in the docstring.

AhmedLeithy commented:
> curious: I see some recorded test with renames and other that were deleted and generated again. I wonder why the combination?

I had initially tried to rerun all the tests, but there were a few that didn't succeed (because of the Azure Active Directory issue), so I renamed the ones that were left over. 

But yea, I agree, it's a bit weird and inconsistent. I've renamed all of them instead.
